src/pages/PrivateEntries/PrivateEntriesAuthenticate.tsx

```diff
@@ -47,8 +54,8 @@ export default function PrivateEntriesAuthenticate({ hasError, setHasError, setR
             Journals are private to you and you only.
           </p>
         </Column>
-        <TextArea
-          title="Password:"
+        <TextField
```

Comment:
Make sure to change this to `TextArea` after fixing that component.

Reply:
like rename it? I feel like TextField is a better description of the component than TextArea?